<?xml version="1.0"?>
<launch>
    <param name="use_sim_time" value="true"/>
    <rosparam command="load" file="$(find bag_tumrgbd)/config/topic_seting.yaml" />

	<node pkg="bag_tumrgbd" type="bag_tumrgbd_converter" name="bag_tumrgbd" output="screen">
		<!-- <rosparam file="$(find bag_tumrgbd)/config/camera_pinhole.yaml" /> -->
	</node>
	
    <!-- <node pkg="image_transport" type="republish" name="republish" args="compressed in:=/left_camera/image raw out:=/left_camera/image" output="screen" respawn="true"/> -->

</launch>